File: ejemplosClase/python/Time.py
## Display the time in CDMX
import time ## imports the package
import math ## Math package

timestamp = time.time() ## Seconds since 01/01/1970 (with decimals)

timestamp -=60*60*5
seconds = math.floor(timestamp%60)
minutes = math.floor(timestamp / 60 %60)
hours = math.floor(timestamp/(60*60) % 24)

print(hours,":",minutes,":",seconds)

File: ejemplosClase/src/ejemplosClase/Time.java
package ejemplosClase;
public class Time {

	public static void main(String[] args) {
		long totalMilliseconds = System.currentTimeMillis();
		
		totalMilliseconds -=1000*60*60*5;
		long seconds = totalMilliseconds / 1000 % 60;
		long minutes = totalMilliseconds / 1000 / 60 % 60;
		long hours 	 = totalMilliseconds / (1000 * 60 * 60) % 24;
		
		System.out.printf("It is %02	d:%02d:%02d (Mexico City Time)\n",hours,minutes,seconds);
		
		
		

	}

}
